src/sentry/sentry_metrics/utils.py

```diff
@@ -14,17 +15,17 @@ class MetricIndexNotFound(InvalidParams):  # type: ignore
     pass
 
 
-def reverse_resolve(index: int) -> str:
+def reverse_resolve(use_case_id: UseCaseKey, index: int) -> str:
```

Comment:
To me it seems strange to have the `use_case_id` be the first argument, can you just make it the last?
In this entire module I mean